src/openrct2/windows/Intent.h

```diff
@@ -42,15 +36,15 @@ class Intent
     Intent * putExtra(uint32 key, sint32 value);
     Intent * putExtra(uint32 key, utf8string value);
 };
+#else
+typedef void Intent;
```

Comment:
Oh, I only just noticed… you have a class `Intent` above that you want to use as opaque pointer in C code? Such case could use a comment explaining things.

I'll need to check if that's the best/safe way to do this.